mod_lookup_identity.c

```diff
@@ -88,6 +99,19 @@ static int lookup_user_by_certificate(request_rec * r) {
 	dbus_error_init(&error);
 	DBusMessage * message = NULL;
 	DBusMessage * reply = NULL;
+#ifndef NO_CERTIFICATE_MAPPING_SUPPORT
+	const char * username;
+	apr_table_t * arg_table = NULL;
+
+	if (cfg->arg_name) {
+		ap_args_to_table(r, &arg_table);
+		if ((username = apr_table_get(arg_table, cfg->arg_name)) == NULL) {
+			username = "";
```

Comment:
I can but I still need to replace the NULL with empty string here. Or I can use auxiliary variable and set username only when apr_table_get returns != NULL. Or I can call apr_table_get twice, once to check the result and then once more to assign the value to username. Is any of the options better then the others?